Packs/qualys/Integrations/QualysEventCollector/QualysEventCollector.py

```diff
@@ -74,21 +78,21 @@ def get_user_activity_logs(self, since_datetime: str, max_fetch: int = 0, next_p
 
         return response.text
 
-    def get_host_list_detection(self, since_datetime: str, max_fetch: int = 0, next_page=None) -> Union[str, bytes]:
+    def get_host_list_detection(self, next_page=None) -> Union[str, bytes]:
         """
-        Make a http request to Qualys API to get user activities logs
+        Make a http request to Qualys API to get assets
         Args:
         Returns:
             response from Qualys API
         Raises:
             DemistoException: can be raised by the _http_request function
         """
         self._headers.update({"Content-Type": 'application/json'})
+        since_datetime = arg_to_datetime(ASSETS_FETCH_FROM).strftime(ASSETS_DATE_FORMAT)
```

Comment:
this should be calculated before this function and sent to it as it should also be an argument to the get assets command.